Add Lisbon Hack Week 2018 blog post (#222)
ipfs/blog: content/post/74-lisbon-hack-week.md

---
date: 2019-03-09
url: 74-lisbon-hack-week
tags: hack week ipfs libp2p community researchers
title: 2018 Hack Week - Lisbon
author: André Cruz
---

> In the week of May 21, 2018, we hosted another Hack Week, yet again in the sunny city of Lisbon.

Reviewer: ```suggestion
In the week of May 21, 2018, we hosted another Hack Week in the sunny city of Lisbon.
```

> The goal of this event was for community members and researchers to meet in person, spend quality time together and work towards solving concrete problems. It was incredible to see groups of people coming up with ideas and solutions for problems that had been lingering for some time. The hack spirit quickly spread and fantastic work came out of it.

Reviewer: ```suggestion
The week was great for core developers, contributors and researchers to spend quality time together and work in a fast paced synchronous format towards solving concrete problems. It was incredible to ideas and solutions flourishing for open problems we've been thinking for a while. The hacker spirit was strong throughout the entire week.
```

> The majority of discussions and collaborations were around IPFS, libp2p and applications built on top of them, but we also hosted a [CRDT Research meetup](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTVkko1ZTn-qcGb-n6EqHff) on the second day.

Reviewer 2: ```suggestion
The majority of discussions and collaborations were around IPFS, libp2p and applications built on top of them, but we also hosted a [CRDT Research meetup](https://blog.ipfs.io/67-crdt-research-meetup) on the second day - check out the playlist [here](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTVkko1ZTn-qcGb-n6EqHff).
```

> The majority of discussions and collaborations were around IPFS, libp2p and applications built on top of them, but we also hosted a [CRDT Research meetup](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTVkko1ZTn-qcGb-n6EqHff) on the second day.

Reviewer: Point to the CRDT Research Meetup blog post once is published (making that a dep to this post)

Author: Will update the link once the other blog post is published.

Reviewer 2: done

> We recorded many of the presentations from the event, and compiled them into a [YouTube playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ). You can see them all here:

Reviewer: ```suggestion
Didn't have a chance to be present? No worries, we got you back. Below you can find the list of report back presentations at the end of the week so that you learn about what got developed, designed and decided. [Full playlist playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ), List:

> Didn't have a chance to be present? No worries, we got your back. Below you can find the list of report back presentations at the end of the week so that you learn about what got developed, designed and decided. [Full playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ), List:

Author: @daviddias isn't the  `Full playlist`, `List` kind of strange here?

Reviewer: Please adjust :)

Author: Changed to:

Didn't have a chance to be present? No worries, we got your back. Many of the report back presentations given at the end of the week were record so that you learn about what got developed, designed and decided. We have compiled them into a [YouTube playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ). You can see them all here:

> Didn't have a chance to be present? No worries, we got your back. Many of the report back presentations given at the end of the week were record so that you can learn about what got developed, designed and decided. We have compiled them into a [YouTube playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ). You can see them all here:

Reviewer 2: ```suggestion
Didn't have a chance to be present? No worries, we got your back. Many of the report back presentations were recorded so that you can learn about what got developed, designed and decided. We have compiled them into a [YouTube playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ). You can see them all here:
```

- [Aegir Browser - Hugo Dias](https://youtu.be/SxYx6IiigIc)
- [Building a dApp - André Cruz, André Sousa](https://youtu.be/wKoLS_8pFb0)
- [Demo of Rust-libp2p - Pierre Krieger](https://youtu.be/cMCy7ZfxOtQ)
- [Files Exchange Example - Diogo Silva](https://youtu.be/WartUw3dD5U)
- [Flow Types for IPFS - Volker Mische](https://youtu.be/YPt28cHQwkg)
- [libp2p Project Early Observations - Mike Goelzer](https://youtu.be/WWOZrP-QMXQ)
- [Preview: IPLD Explorer v1 - Oli Evans](https://youtu.be/P9HnZslts6Q)
- [P2P Testing - Mathias Buus](https://youtu.be/rOFHwifVH-I)
- [Peer-Pad: Authentification with Ethereum - Pedro Teixeira, Joao Santos](https://youtu.be/yc10xRifTmQ)
- [Prop Tests - Yoshua Wuyts](https://youtu.be/EPgtvKMF7VQ)
- [RGA @ -CRDT - Pedro Teixeira](https://youtu.be/ON_L_B7OJZM)
- [Service Worker Gateway (IPFS Node Inside a Service Worker) - Vasco Santos](https://youtu.be/yg1yvylXEC0)

> The Hack Week is almost a tradition at Protocol Labs. While we value working remotely, we recognize the value of hacking things together, face-to-face. That's why we will keep doing events like this and we look forward to the next one!

Reviewer: ```suggestion
We hope you enjoy and see you in future Hack Weeks!
```
